dashboard/src/index.css

```
@@ -1,6 +1,14 @@
 @import "tailwindcss";
 @config "../tailwind.config.js";
 
+/* Public-pages brand type (style-guide): geometric display + machine mono.
+   Generates `font-display` and `font-brandmono` utilities; scoped to the
+   marketing pages via wrapper classes, so the app keeps its default type. */
+@theme {
+  --font-display: "Space Grotesk", ui-sans-serif, system-ui, sans-serif;
+  --font-brandmono: "Spline Sans Mono", ui-monospace, "SFMono-Regular", Menlo, monospace;
```

Stylelint (17.12.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)